src/trajectory/util.cpp

```diff
@@ -151,52 +155,60 @@ UniqueInterpolatedPtr concatenate(
 //==============================================================================
 double findTimeOfClosestStateOnTrajectory(
```

Comment:
Hmm, should this function remain in `aikido::trajectory` or would it make sense to have it in `aikido::distance` instead?